filters/magcoordinatealignfilter/magcoordinatealignfilter.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+   @file magMagCoordinateAlignFilter.cpp
+   @brief MagMagCoordinateAlignFilter
+
+   <p>
+   Copyright (C) 2009-2010 Nokia Corporation
+
+   @author Timo Rongas <ext-timo.2.rongas@nokia.com>
+   @author Antti Virtanen <antti.i.virtanen@nokia.com>
```

Comment:
Yes. This file is basically a copy of CoordinateAlignFilter with a datatype that supports magnetometer raw values and calibration level, so they can be passed on through the chain.